modules/administration/pages/ssl-certs-imported.adoc

```diff
@@ -117,13 +124,23 @@ systemctl restart postgresql.service
 spacewalk-service start
 ----
 
-If you are using a proxy, you need to generate a server certificate RPM for each proxy, using their host names and cnames.
+
+
+=== On the Proxy
+
+If you are using a proxy, you need to generate a server certificate RPM for each proxy, using their hostnames and cnames.
 You should use [command]``mgr-ssl-cert-setup`` also on a {productname} Proxy to replace the certificates.
```

Comment:
You should not generate the "RPM". You just need to get the server certificate and key for the proxy.
If the customer generated it on the server, he should do the same for the proxy. If he get them from his own CA, he just need to make the files available on the proxy. After that, they should use `mgr-ssl-cert-setup` similar as on the server as described above.

Reply:
Maybe something like:
If you are using a proxy, you need to generate a server certificate for the proxy on the server and copy the files to the proxy. Alternative you let your own Certificate Authority generate these files and provide them on the proxy. 
After the files are available, use `mgr-ssl-cert-setup` .....

Reply:
ok, now understood, I think. Using your proposal.